### PR TITLE
Fix nxp 25633 bulk add status completed time

### PR DESCRIPTION
**Pull request by bdelbosc**

### CODE REVIEW

File: nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkConstants.java [hunk 1]
```diff
@@ -21,31 +21,37 @@
 /**
  * Bulk constants for serialization.
  *
- * @since 10.2
+ * @since 10.3
  */
 public class BulkConstants {
 
-    // BulkStatus part
+    private BulkConstants() {
```
Comment (covolution) [inline nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkConstants.java]: Why has the private method moved to the top?

File: nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java [hunk 2]
```diff
@@ -76,15 +83,12 @@
     @AvroEncode(using = InstantAsLongEncoding.class)
     protected Instant scrollEndTime;
 
-    @Nullable
-    protected Long processed;
+    @AvroEncode(using = InstantAsLongEncoding.class)
```
Comment (covolution) [inline nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java]: I don't really understand the how the code works but I assume it does!  Here it looks like you are serialising instants as longs with avro?  But elsewhere you have Instants in json, which are serialized as ISO-8601 strings.

File: nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkConstants.java (hunk 1 above)
Comment (bdelbosc) [reply nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkConstants.java]: yeah weird will fix it

File: nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java (hunk 2 above)
Comment (bdelbosc) [reply nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkStatus.java]: personally I would have prefer normal timestamp everywhere, but it makes sense to have instant encoded as timestamp with a long instead a string representation that is much more bigger.

**Pull request merged**